src/main/java/com/list/todo/controllers/TodoListController.java

```diff
@@ -0,0 +1,109 @@
+package com.list.todo.controllers;
+
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.list.todo.entity.TodoList;
+import com.list.todo.security.CurrentUser;
+import com.list.todo.security.UserPrincipal;
+import com.list.todo.services.TodoListService;
+
+@RestController
+@RequestMapping("/api/todolists")
+public class TodoListController {
+
+	private final TodoListService todoListService;
+	
+
+	@Autowired
+	public TodoListController(TodoListService todoListService) {
+		this.todoListService = todoListService;
+	}
+
+	@GetMapping
+	@PreAuthorize("hasRole('ROLE_USER')")
```

Comment:
`@PreAuthorize("hasRole('ROLE_USER')")` can be put on a controller.
also consider `@PreAuthorize("hasAnyRole('ROLE_ADMIN', 'ROLE_USER')")`